nixos/modules/services/cluster/dynomite.nix

```diff
@@ -0,0 +1,60 @@
+{ config, pkgs, lib, ... }: with lib; let
+  cfg = config.services.dynomite;
+  configFile = pkgs.writeText "dynomite.yaml" "${cfg.config}";
+in {
+  options = {
+    services.dynomite = {
+      enable = mkEnableOption "dynomite server";
+      config = mkOption {
+        default = "";
```

Comment:
Generally what we would do here is not include a `default` and then you would not require the `assertion` as the configuration won't build without a value for `config`.